base/indices.jl

```diff
@@ -1,5 +1,11 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    Dims{N}
+
+An `NTuple` of `N` `Int`s used to represent the dimensions
```

Comment:
Perhaps change "An `NTuple` of `N` `Int`s used to ..." to "An alias for `NTuple{N,Int}` used to ..."